client/src/__tests__/Login.test.jsx

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import Login from "../components/Login";
+
+test('NavBar should render links case-insensitively', () => {
+    render(
+    <Router>
+        <Login />
```

Comment:
This is more of a general comment. Do you need to pass down props to this component to get it to work? Think about it!